### PR TITLE
Auto-generate server URLs for openapi

### PR DESCRIPTION
**Pull request by denis-yuen**
* working on #3217

### CODE REVIEW

File: dockstore-webservice/src/main/resources/openapi3/openapi.yaml [hunk 1]
```diff
@@ -14,6 +14,22 @@ info:
     name: Apache License Version 2.0
     url: https://github.com/dockstore/dockstore/blob/develop/LICENSE
   version: 1.9.0-alpha.1-SNAPSHOT
+servers:
+- url: /api
```
Comment (coverbeck) [inline dockstore-webservice/src/main/resources/openapi3/openapi.yaml]: So this one will kick in if we for example create a perf.dockstore.net?

**Comment by denis-yuen** [comment]
> It would be nice, although maybe more work than it's worth, if

Think the comment was cut-off, maybe a follow-up issue?

File: dockstore-webservice/src/main/resources/openapi3/openapi.yaml (hunk 1 above)
Comment (denis-yuen) [reply dockstore-webservice/src/main/resources/openapi3/openapi.yaml]: My belief is yes, the first choice should kick in and work for any openapi.yaml that is hosted on a server at <domain-name>/api/openapi,yaml which it should be (?) for any thing we deploy using our compose setup/cloud formation approach.

**Comment by coverbeck** [comment]
> > It would be nice, although maybe more work than it's worth, if
> 
> Think the comment was cut-off, maybe a follow-up issue?

I had been thinking that the servers part of the openapi.yaml could be generated dynamically at runtime, based on where the server is running, but I think the way you have it with a relative path and known servers is fine. So ignore

**Pull request merged**